style/layout.css

```diff
@@ -42,23 +54,27 @@ footer > ul {
 }
 
 @media only screen and (min-width: 640px) {
-
+  .nav_main_ul {
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+    background-color: #111111;
+  }
+  .icon-menu3 {
+    display: none;
+  }
+  .nav_main:hover ul {
+    display: flex;
+  }
   .aside_right {
     width: 10%;
     margin-left: 90%;
-    display: flex;
-    justify-content: center;
+    display: block;
+    text-align: center;
   }
   .aside_right > ul {
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
     padding: 0px;
   }
-  /*.aside_right > ul > li {
-    background-color: green;
-    width: 16px;
-  }*/
   .icon-twitter {
```

Comment:
You don't need this here if it's not doing anything. It's akin to commented-out code.